content/source/docs/enterprise/saml/identity-provider-configuration-aad.html.md

```diff
@@ -45,13 +45,16 @@ Follow these steps to configure Azure Active Directory (AAD) as the identity pro
 ## Configure Custom Roles for Team Membership Mapping
 
 1. Create teams in TFE as outlined in [TFE Team Membership](./team-membership.html).
-2. Return to the Azure Portal, navigate to the "App Registration" page, and search for the application you created for TFE. Select your app and select "Manifest".
+2. Return to the Azure Portal, navigate to the "App registrations" page, and search for the application you created for TFE in the "Enterprise applications" page. Select your app and in the left sidebar select "Manifest".
     ![Screenshot: AAD App Registration](./images/sso-aad-saml-app-registration.png)
 3. In the manifest editor, locate the "appRoles" block. This is where you will add additional roles that map users and groups to teams in TFE.
     ![Screenshot: Manifest Editor](./images/sso-aad-saml-manifest-approles.png)
-4. The "appRoles" block may contain roles automatically generated by AAD. New roles should be added after the system roles and must contain a unique GUID value for the ID value of the new role. You can use a tool such as [GUID Generator](https://www.guidgenerator.com) to create these GUIDs. Click "Save" to add the roles.
+4. The "appRoles" block may contain roles automatically generated by AAD. Leave the automatically generated role GUIDs with their default values.  New roles should be added after the system roles and must contain a unique GUID value for the ID value of the new role. You can use a tool such as [GUID Generator](https://www.guidgenerator.com) to create the GUIDs for these new roles. Click "Save" to add the roles.
+
+    -> **Note:** You can add as many roles as your organization needs, such as the `site-admins` role. Azure AD will send the value of these roles as the claim value in the SAML response.
```

Comment:
@nfagerlund like this?

<img width="913" alt="Screen Shot 2019-10-03 at 15 09 28" src="https://user-images.githubusercontent.com/94/66100835-d0878e80-e5ef-11e9-97b2-e8fd2237ad22.png">

Reply:
@dylanegan Yeah!!

I'm going to go ahead and merge this, and we can follow up with more adjustments if we feel the urge later.